tests/integration/test_k8s_with_cluster.py

```diff
@@ -43,38 +38,12 @@
 )
 
 
+@mark.usefixtures("add_secrets")
 @mark.usefixtures("setup_cluster")
 def test_workflow_and_service_management_end_to_end_from_cli(
- docker_image: str, ingress_load_balancer_url: str
+    docker_image: str, ingress_load_balancer_url: str
 ):
     try:
-        process_zero = run(
-            ["bodywork", "setup-namespace", "bodywork-test-project"],
-            encoding="utf-8",
-            capture_output=True,
-        )
-        assert f"creating namespace=bodywork-test-project" in process_zero.stdout
-        assert f"creating service-account={BODYWORK_WORKFLOW_SERVICE_ACCOUNT}" in process_zero.stdout
-        assert f"creating cluster-role-binding={BODYWORK_WORKFLOW_CLUSTER_ROLE}" in process_zero.stdout
-        assert process_zero.returncode == 0
-
-        process_one = run(
-            [
-                "bodywork",
-                "secret",
-                "create",
-                f"--namespace=bodywork-test-project",
-                "--name=bodywork-test-project-credentials",
-                "--data",
-                "USERNAME=alex",
-                "PASSWORD=alex123",
-            ],
-            encoding="utf-8",
-            capture_output=True,
-        )
-        assert process_one.stdout is not None
-        assert process_one.returncode == 0
-
         process_two = run(
```

Comment:
This is now `process_one` - these night need a quick and tedious renaming.

Reply:
Cheers I must have forgotten to amend this one.